metastore/validator.go

```diff
@@ -85,10 +85,6 @@ func (v tableSchemaValidatorImpl) validateIndividualSchema(table *common.Table,
 	for columnID, column := range table.Columns {
 		if !column.Deleted {
 			nonDeletedColumnsCount++
-		} else {
-			if creation {
```

Comment:
we dont need to check?

Reply:
it can be valid if the table is migrated from other clusters, or used for bootstraping, and have deleted columns.